.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.16.3-otp-26
```

Comment:
same for this . let s do it in a separate MR with pipeline change above

Reply:
I suggest removing this file, it doesn't bring any value and makes developers install new versions or update this lock each time. It's the job of CI, in my opinion, to ensure the source code works with the supported Elixir and Erlang versions.

Reply:
this is useful for ASDF and provide information of the developping version quickly